File: CSIT111/Lecture_2/Lecture_Labs/Lecturer_Examples/Circle_3.py
import math

class Circle_3:
    def main(self):
        
        # Assign a value to radius
        radius = 123.4567
        
        # Compute its area and its periemter
        area = math.pi * radius * radius
        perimeter = 2.0 * math.pi * radius
        
        # Display the results
        print("Given radius = ", radius)
        print("Area = ", area)
        print("Perimeter = ", perimeter)

circle_3 = Circle_3()
circle_3.main()

File: CSIT111/Lecture_2/Lecture_Labs/Lecturer_Examples/Circle_3.java
// Given a radius of a circle.
// - We wish to compute its area and its perimeter.
// - Display the info.
//
// Explore the use of some methods defined in the Math class
// For example, PI belongs to the Math class
//
// File name: Circle_3.java

class Circle_3
{
	public static void main (String [ ] args)
	{
		// Declare variables
		double radius;
		double area, perimeter;
		
		// Assign a value to radius
		radius = 123.4567;
		
		// Compute its area and its perimeter
		area = Math.PI * radius * radius;
		perimeter = 2.0 * Math.PI * radius;
		
		// Display the results
		System.out.print ("Given radius = ");
		System.out.println (radius);
		
		System.out.print ("Area = ");
		System.out.println (area);
		
		System.out.print ("Perimeter = ");
		System.out.println (perimeter);
	}
}